gtk/src/auto/functions.rs

```diff
@@ -233,7 +249,7 @@ pub fn device_grab_remove<P: IsA<Widget>>(widget: &P, device: &gdk::Device) {
 
 #[doc(alias = "gtk_disable_setlocale")]
 pub fn disable_setlocale() {
-    assert_initialized_main_thread!();
+    skip_assert_initialized!();
```

Comment:
Shouldn't this be `assert_uninitialized`?

Reply:
Oh derp, too much copy pasting in the config, good catch :)

Reply:
Fixed, thanks